mod-orders-storage/examples/routing_list_get.sample

```diff
@@ -1,5 +1,4 @@
 {
-  "id": "c0d13648-347b-4ac9-8c2f-5bc47248b87e",
```

Comment:
get example shuold be deleted. because when retrieving there should be id

Reply:
Correct, missed the idea. Will fix